hydroflow_lang/src/graph/mod.rs

```diff
@@ -345,16 +366,32 @@ impl Ord for PortIndexValue {
     }
 }
 
+impl Display for PortIndexValue {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            PortIndexValue::Int(x) => write!(f, "{}", x.to_token_stream().to_string()),
+            PortIndexValue::Path(x) => write!(f, "{}", x.to_token_stream().to_string()),
+            PortIndexValue::Elided(_) => write!(f, "[]"),
+        }
+    }
+}
+
```

Comment:
Where is this used?

Reply:
It's used when outputting an error message about incorrect ports. It's in `remove_module_boundary`